packages/components/src/Draggable/DraggableComponent.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { DragSource as dragSource, DropTarget as dropTarget } from 'react-dnd';
+import PropTypes from 'prop-types';
+import flow from 'lodash/flow';
+
+/**
+ * This function adds the 'drag & drop' behaviour on the given component.
+ * Its returns a new component which encaspulates the given one.
+ * The new component is draggable and dropppable.
+ * The props of the component must have at least: 'element' (object) and 'extra' (object).
+ * The props 'element' is used as dragged data.
+ * The props 'extra' must provide dnd callback methods (see elementSource and elementTarget).
```

Comment:
`extra` doesn't sound like a good name. Anything can be put on it.
If the dnd callback must be provided they should be in there own object.
Or find another name.

Reply:
done